### PR TITLE
Combat cameraman s5

### PR DESCRIPTION
**Pull request by AdamWaldie**
When merged this pull request will:
Edit the mentioned files.
Create a class for S5 combat camera that will, for any unit with the variable name S5 to S5_4:
- Change unit loadout to be that of a press journalist.
-  Remove unit damage
- Set post/pre unit init.
- Set captive state - invisible to AI
- When in combination with the correct composition allow access to zeus with limited usage (view only)

### CODE REVIEW

**Comment by Jarvvski** [comment]
Nice to see others working here

cc: @AdamWaldie

File: cScripts/Loadouts/CfgLoadouts_S5.hpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    goggles[] = {"rhs_googles_clear"};
+    gps[] = {"ItemGPS"};
+    headgear[] = {"H_PASGT_basic_blue_press_F"};
+    map[] = {"ItemMap"};
+    nvgs[] = {""};
+    watch[] = {"ItemWatch"};
+    insignia[] = {""};
+    preLoadout = " \
+    [(_this select 0), 's3', 2, true, true] call cScripts_fnc_setPreInitPlayerSettings; \
+    (_this select 0) allowDamage false;(_this select 0) setCaptive true;";
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_S5.hpp]:
```(_this select 0) allowDamage false; \
(_this select 0) setCaptive true;";```

File: cScripts/Loadouts/CfgLoadouts_S5.hpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    compass[] = {"ItemCompass"};
+    goggles[] = {"rhs_googles_clear"};
+    gps[] = {"ItemGPS"};
+    headgear[] = {"H_PASGT_basic_blue_press_F"};
+    map[] = {"ItemMap"};
+    nvgs[] = {""};
+    watch[] = {"ItemWatch"};
+    insignia[] = {""};
+    preLoadout = " \
+    [(_this select 0), 's3', 2, true, true] call cScripts_fnc_setPreInitPlayerSettings; \
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_S5.hpp]: setPreInitPlayerSettings will have a new format to fix a bug:
`[(_this select 0), 's5', 2, 2, true] call cScripts_fnc_setPreInitPlayerSettings;`
Is the new format.

File: cScripts/Loadouts/CfgLoadouts_S5.hpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    (_this select 0) allowDamage false;\(_this select 0) setCaptive true;";
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_S5.hpp]: You forgot the the new line after the \

```
(_this select 0) allowDamage false; \
(_this select 0) setCaptive true;";
```

**Pull request merged**